gradle/testing.gradle

```diff
@@ -20,29 +20,58 @@
 
 if (hasProperty('testEnableCoverage')) {
     apply plugin: 'jacoco'
+    jacoco {
+        toolVersion = jacocoVersion
+    }
     tasks.withType(Test) { task ->
         finalizedBy "jacoco${task.name.capitalize()}Report"
     }
 }
 
+tasks.withType(JacocoReport) { task ->
+    task.reports.xml.required = true
+}
+
+def skipFDB = System.getenv('SKIP_FDB_TESTS') != null && System.getenv('SKIP_FDB_TESTS') == 'true'
+def skipSlow = System.getenv('SKIP_SLOW_TESTS') != null && System.getenv('SKIP_SLOW_TESTS') == 'true'
```

Comment:
It might make sense to change these to be different targets, rather then environment variables, but I didn't feel like messing with that right now.